### PR TITLE
docs: improve "about golangci-lint" in README

### PR DESCRIPTION
**Pull request by alexandear**
This PR improves the second sentence in README:
- fixes the grammar `IDE` -> `IDEs`;
- slightly rewrite the whole sentence.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@
 
 ---
 
-`golangci-lint` is a fast Go linters runner. It runs linters in parallel, uses caching, supports `yaml` config, has integrations
-with all major IDE and has dozens of linters included.
+`golangci-lint` is a fast Go linters runner. It runs linters in parallel, uses caching, supports YAML configuration,
+integrates with all major IDEs, and includes dozens of linters.
```
Comment (ldez) [inline README.md:10]: dozens? :thinking: there are more than 110 linters.

Comment (ldez) [reply README.md:10]: maybe you could also rephase that.

Comment (alexandear) [reply README.md]: Agree, changed to "over a hundred".

**Review by ldez — APPROVED** [review]
LGTM

**Pull request merged**